src/Libraries/RevitNodes/Elements/InternalUtilities/ElementWrapper.cs

```diff
@@ -18,399 +18,271 @@ public static class ElementWrapper
         /// <param name="ele"></param>
         /// <param name="isRevitOwned">Whether the returned object should be revit owned or not</param>
         /// <returns></returns>
-        public static Element ToDSType(this Autodesk.Revit.DB.Element ele, bool isRevitOwned)
+        public static Element ToDSType(this DB.Element ele, bool isRevitOwned)
         {
             // cast to dynamic to dispatch to the appropriate wrapping method
-            dynamic dynamicElement = ele;
-            return ElementWrapper.Wrap(dynamicElement, isRevitOwned);
+            //dynamic dynamicElement = ele;
+            //return ElementWrapper.Wrap(dynamicElement, isRevitOwned);
 
-        }
-
-        #region Wrap methods
-
-        public static UnknownElement Wrap(Autodesk.Revit.DB.Element element, bool isRevitOwned)
```

Comment:
We should decide if we want to deprecate all these methods and keep them one release pointing users to the `ToDSType` method.